vaadinfrontend/frontend/themes/datamanager/components/dialog.css

```
@@ -18,6 +19,66 @@ vaadin-dialog-overlay::part(title) {
   margin-inline-start: 0;
 }
 
+/* set the width of the notification */
+.notification-dialog::part(overlay) {
+  width: 36.75rem;
+}
+
+.notification-dialog .content {
+  width: 100%;
+}
+
+/* notification dialog header */
+.notification-dialog > [slot="header"] {
+  display: flex;
+  align-items: center;
+  justify-items: flex-start;
+  gap: var(--lumo-space-s);
+}
+
+.notification-dialog .title {
+  font-family: var(--lumo-font-family);
+  line-height: var(--lumo-line-height-m);
+  font-size: var(--lumo-font-size-xl);
+  margin: 0;
+}
+
+/* icon in header */
+.notification-dialog > [slot="header"] vaadin-icon {
+  width: 1.83rem;
+  height: 1.83rem;
+  margin: 0.46rem;
```

Nit: Vaadin provides default Lumo Icon methods which we are currently employing in other classes. I'd default to those if possible. 
```suggestion
  width: 1.83rem;
  height: 1.83rem;
  margin: 0.46rem;
```

---

Values are taken from the design. There is not vaadin equivalent matching it.